.github/workflows/test-ruby-head.yaml

```diff
@@ -0,0 +1,31 @@
+name: Test
+
+on:
+  schedule:
+    - cron: '11 14 * * 0'
```

Comment:
It would be better to choose a different time than master branch.
BTW I'm not sure whether it happens for this branch or master branch.

Reply:
Oh! I see. I'll consider it.

Reply:
> BTW I'm not sure whether it happens for this branch or master branch.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

It seems that it happens for the default branch.

I am very confused by the system that allows this can be set up per branch...

Reply:
I'm not sure, but I'm starting to think it's correct to set `schedule` only on the default branch (master).

Reply:
Just an idea is remove this for v1.16 branch, because we never support v1.16 against Ruby head.

Reply:
Agree!